src/Behaviors/Meshes/pointerDragBehavior.ts

```diff
@@ -143,7 +143,7 @@ export class PointerDragBehavior implements Behavior<AbstractMesh> {
      * Attaches the drag behavior the passed in mesh
      * @param ownerNode The mesh that will be dragged around once attached
      */

```

Comment:
you need to add the @param predicate .... as a comment to prevent one of the build failure !!!